README.md

```diff
@@ -460,6 +460,28 @@ rescue JWT::InvalidSubError
 end
 ```
 
+### Finding a Key
+
+To dynamically find the secret that was used for signing the JWT, pass a block to the decode block. The block receives headers and the original payload as parameters. It should return with the secret to verify the signature that was used to sign the JWT.
```

Comment:
To also cover the asymmetric algorithms maybe the wording could be `find the key for verifying the JWT signature`.

The key is not always the secret and not always used for signing tokens.